### PR TITLE
Minor cleanups

### PR DESCRIPTION
**Pull request by suhasdeshpande**
moved isDev as mentioned in TODO to constants.js and fixed few es-lint errors that I came across while running the lint.

### CODE REVIEW

File: __tests__/app/main/actions/copy.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import copy from '../../../../app/main/actions/copy';
 
+const { clipboard: cb } = require('electron');
```
Comment (vutran) [inline __tests__/app/main/actions/copy.js]: Aliasing seems a bit unnecessary. Instead, let's just call it directly like so:

```js
const electron = require('electron');

electron.clipboard.writeText();
```

File: __tests__/app/main/actions/openurl.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import openurl from '../../../../app/main/actions/openurl';
 
+const { shell: sh } = require('electron');
```
Comment (vutran) [inline __tests__/app/main/actions/openurl.js]: Same here.

File: app/main/plugins.js [hunk 3]
```diff
@@ -7,10 +7,7 @@ const { api } = require('dext-core-utils');
 const is = require('is_js');
 const MarkdownIt = require('markdown-it');
 const { PLUGIN_PATH } = require('../../utils/paths');
-const { MAX_RESULTS } = require('../constants');
-
-// @TODO: Move to a `constants` file
-const isDev = process.env.NODE_ENV === 'development';
+const { MAX_RESULTS, IS_DEV: isDev } = require('../constants');
```
Comment (vutran) [inline app/main/plugins.js]: We can remove the alias here as well and just update all references below to `IS_DEV`.

**Comment by suhasdeshpande** [comment]
@vutran Done. let me know what you think.

**Comment by vutran** [comment]
Great work. 👍

File: app/constants.js [hunk 4]
```diff
@@ -1 +1,4 @@
-exports.MAX_RESULTS = 20;
+module.exports = {
+  MAX_RESULTS: 20,
+  IS_DEV: process.env.NODE_ENV === 'development',
```
Comment (adnasa) [inline app/constants.js]: 👍

**Pull request merged**